bot.py

```diff
@@ -16,6 +17,57 @@ class MusicBot(discord.Client):
     The main bot functionality
     """
 
+    def __init__(self):
+        self.handlers = {}
+
+        self.register_command("hello", handler = self.hello)
+        self.register_command("countdown", handler = self.countdown)
+        self.register_command("dinkster", handler = self.dinkster)
```

Comment:
https://www.flake8rules.com/rules/E251.html
No space around inline operators

Reply:
authoritarian linter

Reply:
Ran the black auto formater.